stacks_queues/stack.py

```diff
@@ -21,18 +21,19 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
+
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        return self.store.__str__()
```

Comment:
Defining the `__str__` method in a class is what we call operator overloading -- Python will convert any object to a string with the str() method by default, but it may not always be what we want. By defining the `__str__` method, we can overwrite Python's default behavior to behavior that we want. However, when we call the overloaded method on an instance of the class, we do not need the dunders, we can just say `str(object_instance)`

```suggestion
        return str(self.store)
```